drivers/AnalogIn.cpp

```diff
@@ -25,3 +25,6 @@ SingletonPtr<PlatformMutex> AnalogIn::_mutex;
 };
 
 #endif
+
```

Comment:
These changes are only to showcase that the script is running properly - only changed files are being checked.